ui/app/templates/components/tool-unwrap.hbs

```diff
@@ -11,10 +11,7 @@
     <nav class="tabs has-bottom-margin-l">
       <ul>
         {{! template-lint-configure no-nested-interactive "warn" }}
-        <li
-          aria-selected={{if (eq @unwrapActiveTab "data") "true" "false"}}
```

Comment:
This was from ember-template-lint saying `aria-selected` shouldn't be used on an `li`